ogusa/ogusa_default_parameters.json

```diff
@@ -33,29 +33,29 @@
         0.995
     ],
     "sigma": 1.5,
-    "gamma": [0.38081351263182506],
+    "gamma": [0.38],
     "epsilon": [1.0],
     "Z": [[1.0]],
     "delta_annual": 0.05,
     "ltilde": 1.0,
     "world_int_rate_annual": [0.04],
     "initial_foreign_debt_ratio": 0.4,
     "zeta_D": [0.4],
-    "zeta_K": [0.1],
+    "zeta_K": [0.5],
     "tG1": 20,
     "tG2": 256,
-    "alpha_T": [0.06117398512328073],
-    "alpha_G": [0.011231117159158614],
+    "alpha_T": [0.052],
+    "alpha_G": [0.091],
     "alpha_I": [0.0],
     "rho_G": 0.1,
     "debt_ratio_ss": 2.0,
-    "initial_debt_ratio": 0.9315882600432254,
-    "r_gov_scale": [0.8471534653465324],
-    "r_gov_shift": [0.011677698019801785],
+    "initial_debt_ratio": 0.99,
+    "r_gov_scale": [0.0],
+    "r_gov_shift": [-0.005, -0.007, -0.009, -0.01],
     "cit_rate": [[0.21]],
     "c_corp_share_of_assets": 0.55,
-    "adjustment_factor_for_cit_receipts": [0.30909090909090914],
-    "tau_c": [[0.0]],
+    "adjustment_factor_for_cit_receipts": [0.3090],
+    "tau_c": [[0.01]],
```

Comment:
@jdebacker. Why is `tau_c=0.01` in this default parameterization? Is this capturing average state sales tax?

Reply:
@jdebacker. Is this to capture national excise tax revenue?

Reply:
I was basing it on CBO's value of "other" federal revenues, which are mostly excise taxes and tariffs and so I thought we'll modeled by a consumption tax.